.github/workflows/trigger-quantic-e2e-setup.yml

```diff
@@ -0,0 +1,51 @@
+name: Trigger Quantic E2E Setup
```

Comment:
Let's make this a bit more generic  and not make it only about E2E cause this workflow could be used for other things.
something like `Setup Quantic Examples`

Reply:
sure!